test_requirements/requirements_core.txt

```diff
@@ -0,0 +1,3 @@
+requests
+pytest
+narwhals
```

Comment:
Is `plotly` installed by the time these requirements are installed? If so, might it be a good idea to remove `narwhals` here to ensure `narwhals` was correctly installed as a dependency of `plotly`?

Reply:
Ah yeah that's a good idea. @emilykl Do you agree?

Reply:
Ok - I think this change requires merging #5044 because narwhals isn't being installed by default. I'll merge with main once that's in and remove narwhals here!

Reply:
Ok - I think this change requires merging #5044 because narwhals isn't being installed by default. I'll merge with main once that's in and remove narwhals here!

Reply:
Done!